src/main/java/frc/robot/subsystems/drivetrain/commands/DriveForward.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.OI;
+import frc.robot.Robot;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+import frc.robot.valuetuner.WebConstant;
+import org.techfire225.webapp.FireLog;
+
+public class DriveForward extends CommandBase {
+
+    private SwerveDrive swerveDrive;
+    private WebConstant target = new WebConstant("targetSpeed", 0);
+
+    public DriveForward(SwerveDrive swerveDrive) {
+        this.swerveDrive = swerveDrive;
+        addRequirements(swerveDrive);
+    }
+
+    @Override
+    public void initialize() {
+    }
+
+    @Override
+    public void execute() {
+        double forward = -OI.getJoystickY();
+        forward = joystickDeadband(forward);
```

Comment:
You can combine this into 1 line
```suggestion
        double forward = joystickDeadband(-OI.getJoystickY());
```